packages/components/src/modal/style.scss

```diff
@@ -146,4 +146,25 @@
 		outline: 2px solid transparent;
 		outline-offset: -2px;
 	}
+
+	@include break-small() {
```

Comment:
Is there a reason why these styles are being applied to `components-modal__content` instead of `components-modal__frame` ?

I think it would be good to have full screen styles and "content-width" styles applied to the same element and in the same place in the styles file, since they all affect the same properties (width and max-width)